Ice/Settings/SettingsWindow.swift

```diff
@@ -16,9 +16,28 @@ struct SettingsWindow: Scene {
                 .onAppear(perform: onAppear)
                 .environmentObject(appState)
                 .environmentObject(appState.navigationState)
+                .background(WindowAccessor { window in
+                    window.level = .floating
+                })
```

Comment:
We have a custom modifier, `View.readWindow(onChange:)`, so let's use that here instead.

```suggestion
                .readWindow { window in
                    window.level = .floating
                }
```